### PR TITLE
feat: change to real value for api

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Set real value to
  1) Dashboard Variables(2 variables)
  스크린샷 2023-06-09 오후 5 23 10
  2) Widget Group By (4 options)
  스크린샷 2023-06-09 오후 5 21 42

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue [hunk 1]
```diff
@@ -184,10 +187,17 @@ const handleUpdateSearchText = debounce((text: string) => {
...
-const loadSearchResourceOptionsByCostAnalysis = async () => {
+const loadSearchResourceOptions = async () => {
     if (state.variableProperty.options?.type === 'SEARCH_RESOURCE') {
+        // NOTE: Some variables(asset) require specific API filters.
+        const apiQueryHelper = new ApiQueryHelper();
+        if (state.variableProperty.name === ASSET_REFERENCE_TYPE_INFO.asset_compliance_type.name) {
+            apiQueryHelper.addFilter({ k: 'labels', o: '=', v: 'Compliance' });
+        } else if (state.variableProperty.name === ASSET_REFERENCE_TYPE_INFO.asset_account.name) {
+            apiQueryHelper.addFilter({ k: 'provider', o: '=', v: 'aws' });
+        }
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue:198]: description: This code is for this.

스크린샷 2023-06-09 오후 5 23 10

File: apps/web/src/services/dashboards/widgets/_configs/config.ts [hunk 2]
```diff
@@ -53,9 +53,13 @@ export const ASSET_GROUP_BY = {
...
     // asset reference
-    COMPLIANCE_NUMBER: ASSET_REFERENCE_TYPE_INFO.asset_compliance_number.key,
+    REQUIREMENT_ID: ASSET_REFERENCE_TYPE_INFO.asset_requirement_id.key,
     SERVICE: ASSET_REFERENCE_TYPE_INFO.asset_service.key,
-    COMPLIANCE_TYPE: ASSET_REFERENCE_TYPE_INFO.asset_compliance_type.key,
+    /* HACK:
+    *   The group_by originally matched the reference_key(for autocomplete), but the COMPLIANCE_TYPE is different.
+    *   For now, it is hardcoded, but there's a need to manage it through config in the future.
+    */
+    COMPLIANCE_TYPE: 'cloud_service_type',
```
Comment (yuda110) [inline apps/web/src/services/dashboards/widgets/_configs/config.ts:62]: description: key for autocomplete(select dropdown) is `name`, but key for group_by is `cloud_service_type`😭

File: apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue [hunk 3]
```diff
@@ -184,10 +187,17 @@ const handleUpdateSearchText = debounce((text: string) => {
     reloadMenu();
 }, 200);
 
-const loadSearchResourceOptionsByCostAnalysis = async () => {
+const loadSearchResourceOptions = async () => {
     if (state.variableProperty.options?.type === 'SEARCH_RESOURCE') {
+        // NOTE: Some variables(asset) require specific API filters.
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue:192]: praise: thanks for the comment!

File: apps/web/src/lib/reference/asset-reference-config.ts [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
+        key: 'data.requirement_id',
+        name: 'Requirement ID',
     },
     asset_service: {
         type: 'asset_service',
-        key: 'service',
+        key: 'data.service',
         name: 'Service',
     },
 } as const;
```
Comment (sulmoJ) [inline apps/web/src/lib/reference/asset-reference-config.ts:22]: praise: I believe you

File: apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/dashboard-customize/modules/VariableSelectorDropdown.vue:198]: Thanx for explanation!

File: apps/web/src/services/dashboards/widgets/_configs/config.ts (hunk 2 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_configs/config.ts:62]: Oh, I think it doesn't matter. That's why we manage group_by config independently.

**Review by WANZARGEN — APPROVED** [review]
LGTM!!!

**Review by sulmoJ — APPROVED** [review]
I only checked the code conventions!
LGTM!

**Pull request merged**